### PR TITLE
Tps extension

### PR DESCRIPTION
**Pull request by ndem0**

### CODE REVIEW

**Comment by ndem0** [comment]
Deat @giadadalla,  I reopened your PR with the new changes.
I'll add a couple of comments about minor fix that we ask you. Thanks for your effort

File: pygem/rbf_factory.py [hunk 1]
```diff
@@ -77,23 +77,26 @@ def inv_multi_quadratic_biharmonic_spline(X, r=1):
...
         .. math::
             \\varphi(\\boldsymbol{x}) =
-            \\left(\\frac{\\boldsymbol{x}}{r}\\right)^2
+            \\left(\\frac{\\boldsymbol{x}}{r}\\right)^k
             \\ln\\frac{\\boldsymbol{x}}{r}
 
         :param numpy.ndarray X: the norm x in the formula above.
-        :param float r: the parameter r in the formula above.
+        :param float r: the parameter r in the formula above. 
+
```
Comment (ndem0) [inline pygem/rbf_factory.py:91]: the description of parameter `k` is missing (see line before as example)

File: pygem/rbf_factory.py [hunk 2]
```diff
@@ -77,23 +77,26 @@ def inv_multi_quadratic_biharmonic_spline(X, r=1):
...
 
         :param numpy.ndarray X: the norm x in the formula above.
-        :param float r: the parameter r in the formula above.
+        :param float r: the parameter r in the formula above. 
+         
 
         :return: result: the result of the formula above.
         :rtype: float
+
+        With k=2 the function is "radius free", that means independent of radius value.
```
Comment (ndem0) [inline pygem/rbf_factory.py:96]: move before parameter description

File: pygem/rbf_factory.py [hunk 3]
```diff
@@ -77,23 +77,26 @@ def inv_multi_quadratic_biharmonic_spline(X, r=1):
...
         """
         arg = X / r
-        result = arg * arg
+        result = np.power(arg,k)
```
Comment (ndem0) [inline pygem/rbf_factory.py:99]: please add a blank space after the comma

**Comment by giadadalla** [comment]
@ndem0 I changed what you asked me. Let me know if everything is ok

**Comment by mtezzele** [comment]
> @ndem0 I changed what you asked me. Let me know if everything is ok

Thank you for the contribution. Please move the comment regarding `k=2` before the description of the parameters and after the formula. Keep the empty lines since otherwise the documentation will not be formatted properly.

**Pull request merged**